build/Tests/UnitTests/AudioToolbox/AudioToolbox.UnitTests.vcxproj

```diff
@@ -135,7 +135,7 @@
     <Link>
       <SubSystem>Console</SubSystem>
       <GenerateDebugInformation>true</GenerateDebugInformation>
-      <AdditionalDependencies>;libdispatch.lib;mincore.lib;%(AdditionalDependencies)</AdditionalDependencies>
+      <AdditionalDependencies>AudioToolbox.lib;mfuuid.lib;mfplat.lib;libdispatch.lib;mincore.lib;%(AdditionalDependencies)</AdditionalDependencies>
```

Comment:
I wasnt getting any build errors and confirmed that again now with a new clone and build. 
Without the dependency on AudioToolbox.lib (or with dependency on AudioToolboxLib.lib) I am getting an unresolved external symbol error for the AudioConverterNew and AudioConvertDispose calls